R/readepi.R

```diff
@@ -29,8 +29,8 @@
 #' @returns a `list` of 1 or several object(s) of type `data frame`.
 #' @export
 readepi <- function(data_source = NULL,
```

Comment:
A good test to verify the consistency in the way your internal are set up would be to simplify this `readepi()` function by removing the `if`/`else` and just have a `switch()` that dictates the function call. Then, all argument are passed to all functions indifferently and everything has to work out of the box.

Reply:
this is accounted account for at commits `85a935ac` and `3ab5e778`